src/components/Settings/SettingsNetwork/index.tsx

```diff
@@ -44,9 +38,19 @@ const messages = defineMessages('components.Settings.SettingsNetwork', {
     "Use ',' as a separator, and '*.' as a wildcard for subdomains",
   proxyBypassLocalAddresses: 'Bypass Proxy for Local Addresses',
   validationProxyPort: 'You must provide a valid port',
+  advancedNetworkSettings: 'Advanced Network Settings',
+  networkDisclaimer:
+    'Network parameters from your container/system should be used instead of these settings. See the {docs} for more information.',
+  docs: 'documentation',
+  forceIpv4First: 'IPv4 Resolution First',
```

Comment:
Wouldn't it be better to have `Force IPv4 Resolution First`? So it's more descriptive?

Reply:
I'm not sure it would make a difference, but I can change it if you want.

Reply:
It makes sense more because thats what it's referred to everywhere else (code + node docs)